src/config/chain_config.js

```diff
@@ -60,6 +60,7 @@ class ChainConfig {
         EIP20CoGateway: this.auxiliaryCoGatewayAddress,
         Anchor: this.auxiliaryAnchorAddress,
         UtilityToken: this.auxiliaryUtilityTokenAddress,
+        OSTPrime: this.auxiliaryOSTPrimeAddress,
```

Comment:
OST' on 200 is 0x05cd5fcd2aeca6aea1a554fae9fac76ce52dc5d6
can we add this to constructor of chainconfig as a default value just now?